lib/Elements/Math.tsx

```
@@ -0,0 +1,229 @@
+import React, { memo, useEffect, useMemo, useRef, useState } from "react";
+import { View, Text } from "react-native";
+import { useStreamDown } from "../context";
+import type { MathToken } from "../marked/marked-math";
+import WebView from "react-native-webview";
+
+interface MathProps {
+  token: MathToken;
+}
+
+export const Math = memo<MathProps>(({ token }) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Math" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)